### ISSUE TITLE
Toggle does not support aria-label correctly

### ISSUE THREAD

**Issue by connorjsmith** [microsoft/fast-dna#1834]
**Describe the bug; what happened?**
Assigning an `aria-label` to a `<Toggle>` makes its underlying `div` scannable in Microsoft Narrator.

**What are the steps to reproduce the issue?**
Assign an aria-label to a Toggle component. This aria-label is assigned to the outer div wrapping the input, making this div scannable when it shouldn't be.

**What behavior did you expect?**
The aria-label should be applied to the input, ensuring the wrapping div does not become scannable.

**If applicable, provide screenshots:**
Scan mode finding the outer div
![image](https://user-images.githubusercontent.com/5892363/59119176-f2edf180-8906-11e9-9357-a0c0ffa67723.png)
Caps + Right finds the inner input element
![image](https://user-images.githubusercontent.com/5892363/59119198-000ae080-8907-11e9-893c-9447b028576d.png)

**In what environment did you see the issue?**

* OS & Device: Windows 10 PC
* Browser: Edge Canary
* Version: 76.0.182.0

**Comment by nicholasrice** [comment]
We have this same issue with radio and checkbox - unhandled props are passed to the outer div and not the input (which is generally where developers want unhandled props to go). Unfortunately this change would be breaking, but I do think we want to make the change.

In this particular case, why is the aria-label necessary? What does it do that a normal label element for the control doesn't?

**Comment by connorjsmith** [comment]
The included label is a bit too limiting for our designs, so we are forced to use a different label (with rich text) and use the plaintext content as an aria-label for the toggle
